README.md

```diff
@@ -1,18 +1,18 @@
 ## grammY Commands Plugin
 
-This plugin provides a convenient way to define and manage commands for your grammY bot. It simplifies the process of setting up commands with scopes and localization.
```

Comment:
Have you considered writing markdown files with one sentence per line? I think we should do it that way for this repo, too, and since you're reformatting the file anyway, we might just as well do it right now.

Reply:
This is placeholder text. It needs to be replaced by better docs. But I can do one sentence per line anyway.